tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -127,31 +127,23 @@ stages:
               mv $(testFilesPath)/src/test/* ${{ variables.testWorkspace }}/node_modules/${{ testPackage }}/src/test
 
         # Install package dependencies (this gets devDependencies installed so we can then run the package's tests)
-        # and run tests.
         - task: Bash@3
-          displayName: Run execution-time tests - ${{ testPackage }}
+          displayName: Install dependencies - ${{ testPackage }}
           inputs:
             targetType: 'inline'
             workingDirectory: ${{ variables.testWorkspace }}/node_modules/${{ testPackage }}
             script: |
-              echo "Run execution-time tests for ${{ testPackage }}"
               cp ${{ variables.testWorkspace }}/.npmrc . ;
-
-              # Hacky workaround to prevent issues because nodegit fails to install in Node 18.
-                # good-fences is the one that transitively brings in the dependency on nodegit, so we remove it from
-                # the package whose tests we're going to run if it's there (we don't need it to run the tests).
-                # TODO: remove this once the packages for which we run perf tests don't depend on good-fences anymore
-                # or we have a better solution.
-                GOOD_FENCES_FOUND=$(cat package.json | grep "good-fences" | wc -l)
-                if [[ $GOOD_FENCES_FOUND == 1 ]]; then
-                  echo "good-fences found in package.json for '${{ testPackage }}'. Removing it to avoid nodegit install issues."
-                  npm uninstall good-fences;
-                  echo "Deleting fence.spec.js files that use good-fences."
-                  # 'rm **/fence.spec.js' works in zsh but not bash
-                  find . -type f -name 'fence.spec.js' -delete;
-                fi
-
               npm i;
+
+        # Run tests
+        - task: Bash@3
```

Comment:
Nit: Can we use `Npm@1` for this?

We could also split the install step into 2 tasks to use the npm task for install, but that might be overkill.

Reply:
Although, actually, splitting the install step might be nice too. Then we can use the [CopyFiles](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/copy-files-v2?view=azure-pipelines&tabs=yaml) task for the npmrc copy.

Reply:
Done!